templates/zerver/plans.html

```diff
@@ -17,28 +17,26 @@
     <div class="main">
         <div class="pricing-model">
             <div class="padded-content">
-                <h1>Get started with Zulip today</h1>
+                <h1>Get started today</h1>
 
                 <div class="pricing-container">
                     <div class="block">
                         <div class="plan-title use-our-servers">
```

Comment:
Should this be updated to zulip-cloud as well? And also in the landing-page.css.

Reply:
good catch, thanks!